docs/prover/cli/options.md

```diff
@@ -793,29 +793,117 @@ Advanced options that control control flow splitting
 
 See [here](control-flow-splitting) for an explanation of control flow splitting.
 
+(-depth)=
+### `--prover_args '-depth <number>'`
+
+**What does it do?**
+
+Sets the maximum splitting depth.
+
+**When to use it?** 
+
+When the deepest splits are too heavy to solve, but not too high in number,
+increasing this will lead to smaller, but more split leafs, which run at the
+full SMT timeout (as set by {ref}`--smt_timeout`). Conversely, if run time is
+too high because there are too many splits, decreasing this number means that
+more time is spent on fewer, but bigger split leafs.
+
+**Example**
+
+```sh
+certoraRun Bank.sol --verify Bank:bank.spec --prover_args '-depth 5'
+```
 
 (-mediumTimeout)=
 ### `--prover_args '-mediumTimeout <seconds>'`
 
-The "medium timeout" determines how much time is given to checking a split at
-not max-depth before we split again.
+The "medium timeout" determines how much time the SMT solver gets for checking a
+split that is not a split leaf. 
+
+**What does it do?**
+
+Sets the time that non-leaf splits get before being split again. 
+
+**When to use it?** 
+
+When a little more time can close some splitting subtrees early, this can save a
+lot of time, since the subtree's size is exponential in the remaining depth. On
+the other hand, if something will be split further anyway, this can save the
+run time spent on intermediate "TIMEOUT" results. Use
+{ref}`-smt_initialSplitDepth` to eliminate that time investment altogether up to
+a given depth.
+
+**Example**
+
+```sh
+certoraRun Bank.sol --verify Bank:bank.spec --prover_args '-mediumTimeout 20'
+```
+
+
+
+% The regular SMT timeout determines how much time is maximally spent on checking
```

Comment:
not sure why this is commented out

Reply:
removed and added an extra sentence to the description of `-mediumTimeout` right above